components/brave_vpn/browser/connection/win/brave_vpn_helper/BUILD.gn

```diff
@@ -0,0 +1,103 @@
+# Copyright (c) 2022 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import("//build/toolchain/win/midl.gni")
+import("//build/util/process_version.gni")
+import("//chrome/process_version_rc_template.gni")
+
+assert(is_win)
+
+# Must be in a config because of how GN orders flags (otherwise /W4 will appear
+# after this, and turn it back on).
+config("no-missing-braces") {
+  # Macros invoked by WRL CoCreatableClass omit optional braces.
+  cflags = [ "-Wno-missing-braces" ]
+}
+
+source_set("common") {
+  sources = [
+    "brave_vpn_helper_constants.h",
+    "brave_vpn_helper_state.cc",
+    "brave_vpn_helper_state.h",
+    "scoped_sc_handle.h",
+  ]
+  deps = [ "//base" ]
+}
+
+source_set("lib") {
+  visibility = [
+    ":brave_vpn_helper",
+    ":unit_tests",
+  ]
+  sources = [
+    "brave_vpn_dns_delegate.h",
+    "vpn_dns_handler.cc",
+    "vpn_dns_handler.h",
+    "vpn_utils.cc",
+    "vpn_utils.h",
+  ]
+
+  deps = [
+    ":common",
+    "//base",
+    "//brave//components/brave_vpn/browser/connection/win:utils",
+    "//brave//components/brave_vpn/common",
+    "//third_party/abseil-cpp:absl",
+  ]
+
+  libs = [
+    "Fwpuclnt.lib",
+    "Iphlpapi.lib",
```

Comment:
I think we need to add `"rasapi32.lib` here as vpn_utils.cc uses ras pi?

Reply:
done